src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationSchedule.kt

```diff
@@ -71,4 +71,10 @@ class LocationSchedule(
   override fun toString(): String = this::class.simpleName +
     "(locationScheduleId = $locationScheduleId, startDay = $startDayOfWeek, endDay = $endDayOfWeek " +
     "startTime = $startTime, endTime = $endTime)"
+
+  // TODO
+  fun isAvailableFor(probationTeam: ProbationTeam, onDateTime: LocalDateTime) = true
```

Comment:
@timharrison-moj I am unclear how this will work at the moment.  We need to discuss this further.